### DISCUSSION TITLE
What icon images should be provided for an ItemTemplate?

### DISCUSSION THREAD

**Discussion by mrlacey** [VsixCommunity/Community.VisualStudio.Toolkit#35] [Q&A] [answered]
I know @madskristensen's advice for the image for a VSIX (90x90 PNG file) but what about for an item template?
The default templates include a .ico file containing 9 different size versions of the image. Surely these aren't all needed. Does anyone know for sure?
Would a version of the ItemTemplate project with correctly sized (& format) be a useful inclusion in the community toolkit?

**Comment by madskristensen** [comment, marked as answer]
I use 64x64 pixel PNG files for item templates
